bin/gollum

```diff
@@ -121,10 +121,13 @@ MSG
     wiki_options[:allow_uploads]    = true
     wiki_options[:per_page_uploads] = true if mode == :page
   end
-  opts.on('--mathjax', 'Enable MathJax (renders mathematical equations).',
-    'By default, uses the \'TeX-AMS-MML_HTMLorMML\' config with the \'autoload-all\' extension.') do
-    wiki_options[:mathjax] = true
-    wiki_options[:mathjax_config] = 'mathjax.config.js'
+  opts.on('--math [RENDERER]', [:mathjax, :katex], 'Enable rendering of mathematical equations.',
+    'Valid renderers: mathjax, katex. Default: mathjax.') do |mode|
```

Comment:
Have we discussed enabling `katex` as the default option? Given that it is easier to bundle and to update, this may be preferable for the majority of users?

Reply:
That sounds great. We just may want to make it very explicit that MathJax and KaTeX are not fully compatible with each other (to my limited knowledge, at least). So if someone is already using MathJax they may want to continue using MathJax.

Reply:
Yes, especially since Github and Gitlab also diverge in this respect. So for the time being I think we want to offer both. But given how `--mathjax` and the corresponding config setting will remain functional for `6.x`, I think those existing users shouldn't be bothered if we default the new `--math` flag to `katex`, right?

Reply:
Right!

Reply:
Great idea! An even more radical solution would be to activate `katex` out of the box, so without the need for a switch, and then let users override it with `--mathjax`? I'm not sure if there are downsides to enabling math support out of the box (potentially page load times, I suppose).